lib/controller/WebInitController.js

```diff
@@ -1,32 +0,0 @@
-import StudyRequestReasonDAO from '@/lib/db/StudyRequestReasonDAO';
```

Comment:
...we might actually still want this around `study_request_assignees`, since those *should* be dynamic.

Reply:
(going to fix in a later revision - need to ask @shinechaudhuri how assignment is being handled here)